### PR TITLE
Update october 2018

### PR DESCRIPTION
**Pull request by ConnorHeidema**
Updated the gradle file, and updated variable names

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (keeferrourke) [inline .idea/codeStyles/Project.xml]: Probably don't need to commit this file.

File: app/build.gradle [hunk 2]
```diff
@@ -1,83 +1,85 @@
...
+        /*
+            rightmesh_artifactory_username=<username>
+            rightmesh_artifactory_password=<hash of your developer portal password>
+            rightmesh_reflect_key=<some key that you have access to>
+        */
+        artifactory_app_username = "${rightmesh_artifactory_username}"
+        artifactory_app_password = "${rightmesh_artifactory_password}"
+        artifactory_app_key = "${rightmesh_reflect_key}"
+
+        maven_repo = "https://research.rightmesh.io/artifactory/libs-local"
```
Comment (keeferrourke) [inline app/build.gradle]: This should be `https://artifactory.rightmesh.io/artifactory/maven`

File: app/build.gradle [hunk 3]
```diff
@@ -1,83 +1,85 @@
...
+    }
+}
+
 android {
     compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    buildToolsVersion '27.0.3'
     defaultConfig {
-        applicationId "io.left.reflect"
+        applicationId "io.left.hellomesh"
```
Comment (keeferrourke) [inline app/build.gradle]: This is the `reflect` repo, the applicationId should still be `io.left.reflect`

**Review by keeferrourke — CHANGES_REQUESTED** [review]
I noticed just a few minor issues that should be addressed before this is merged :sweat_smile:

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (FrazerSeymour) [reply .idea/codeStyles/Project.xml]: Yeah in general we don't need `.idea` files at all.

File: app/build.gradle [hunk 4]
```diff
@@ -1,83 +1,85 @@
...
-            url "https://research.rightmesh.io/artifactory/libs-release-local"
+            url maven_repo
             credentials {
                 username artifactory_app_username
                 password artifactory_app_password
             }
         }
-
-        // For RetroLambda.
+        google()
```
Comment (FrazerSeymour) [inline app/build.gradle]: Do we need all of these repos anymore if we don't need RetroLambda?

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (ConnorHeidema) [reply .idea/codeStyles/Project.xml]: Kept all .idea files the way they were before.

File: app/build.gradle (hunk 2 above)
Comment (ConnorHeidema) [reply app/build.gradle]: Changed

File: app/build.gradle (hunk 3 above)
Comment (ConnorHeidema) [reply app/build.gradle]: Moved back to reflect

File: app/build.gradle (hunk 4 above)
Comment (ConnorHeidema) [reply app/build.gradle]: Removed unnecessary repos

**Review by keeferrourke — APPROVED** [review]

**Pull request merged**